CRM/Utils/Mail.php

```diff
@@ -184,14 +184,20 @@ public static function send(array &$params): bool {
       return FALSE;
     }
 
-    $textMessage = $params['text'] ?? NULL;
-    $htmlMessage = $params['html'] ?? NULL;
-    $attachments = $params['attachments'] ?? NULL;
-
-    // CRM-6224
-    if (trim(CRM_Utils_String::htmlToText($htmlMessage)) == '') {
+    $htmlMessage = $params['html'] ?? FALSE;
+    if (trim(CRM_Utils_String::htmlToText((string) $htmlMessage)) === '') {
       $htmlMessage = FALSE;
     }
+    $attachments = $params['attachments'] ?? NULL;
+    if (!empty($params['text'])) {
```

Comment:
This means you wouldn't be able to explicitly set the text to the empty string, but I can't think many people are currently doing that.

Reply:
@demeritcowboy yeah my thinking is there isn't much use-case for it